extensions/amp-analytics/0.1/linker-manager.js

```diff
@@ -273,7 +273,7 @@ export class LinkerManager {
   isDomainMatch_(hostname, domains) {
     // If given domains, but not in the right format.
     if (domains && !Array.isArray(domains)) {
-      user().warn(TAG, `${name} destinationDomains must be an array.`);
```

Comment:
LGTM. What is the name from `maybeAppendLinker_` should we include that piece of information for easier debugging?

Reply:
Yeah thats what it was before, the problem is that a form can also now call this and we don't have the name there.

Reply:
Actually I'm wrong, I will change this to pass name along.